servers/rendering/shader_compiler.cpp

```diff
@@ -1583,6 +1596,19 @@ void ShaderCompiler::initialize(DefaultIdentifierActions p_actions) {
 	texture_functions.insert("textureQueryLod");
 	texture_functions.insert("textureQueryLevels");
 	texture_functions.insert("texelFetch");
+
+	// global functions
```

Comment:
```suggestion
	// Global functions.
```
Just a little comment nitpick